pkg/elementalConfig/config.go

```diff
@@ -369,6 +369,14 @@ func NewUpgradeSpec(cfg v1.Config) (*v1.UpgradeSpec, error) {
 		}
 	}
 
+	// If we have oem in the system
+	if ep.OEM != nil {
+		// But it has no mountpoint
+		if ep.OEM.MountPoint == "" {
```

Comment:
since both come from `ep.OEM`, it would be nice to have a predicate function for this e.g. `ep.MissingMountPoint()`

Reply:
done!